x-pack/plugins/profiling/public/components/flame_graphs_view/get_information_rows.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { NOT_AVAILABLE_LABEL } from '../../../common';
+import { describeFrameType } from '../../../common/profiling';
+
+export function getInformationRows({
+  fileID,
+  frameType,
+  exeFileName,
+  addressOrLine,
+  functionName,
+  functionOffset,
+  sourceFileName,
+  sourceLine,
+}: {
+  fileID: string;
+  frameType: number;
+  exeFileName: string;
+  addressOrLine: number;
+  functionName: string;
+  functionOffset: number;
+  sourceFileName: string;
+  sourceLine: number;
+}) {
+  const executable = fileID === '' && addressOrLine === 0 ? 'root' : exeFileName;
+  const functionSourceLine = functionOffset > 0 ? ` (+${sourceLine - functionOffset})` : '';
```

Comment:
I am not convinced that the relative is of any use here. E.g. `+66` means 66 lines inside the function - to navigate there you have to manually calculate the absolute line number.
Also: since the aggregation ignores this line, the value is an arbitrary value.

IMO better: show the function's line number after the source file name, e.g. "xyz.py#123"

Reply:
I agree about the relative line number. `prodfiler` used the relative line number in the information window, yet the function line number in the flamegraph labels. Your last suggestion would make the flamegraph label and the information window agree, which is what I decided to go with.